shared/chat/conversation/header-area/normal/index.native.js

```diff
@@ -28,8 +28,13 @@ const Wrapper = (props: {
 }) => (
   <Box2 direction="horizontal" style={styles.container}>
     <ClickableBox onClick={props.onBack} style={styles.leftMargin}>
-      <Icon type="iconfont-arrow-left" fontSize={24} color={globalColors.black_40} />
-      {!!props.badgeNumber && <Badge badgeNumber={props.badgeNumber} badgeStyle={styles.badge} />}
+      <Icon
```

Comment:
I noticed a lot of similarity between this back button and the back button in common adaptors – maybe we want to refactor and get rid of duplicate code in a future ticket?